hello_bot.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import sys
+
+
+def send_message(token, id, text):
+    response = requests.post(f'https://api.telegram.org/bot{token}/sendMessage', json={'chat_id': id, 'text': text})
+    return response.status_code == 200
+
+
+def get_message(token, offset):
+    response = requests.get(f'https://api.telegram.org/bot{token}/getUpdates', params={'offset': offset,'timeout': 60})
+    return response.json()
```

Comment:
В `send_message` возвращается `true` если код ответа 200.

А здесь без какой-либо проверки предполагается, что всё хорошо и парсится JSON.

> In case the JSON decoding fails, `r.json()` raises an exception. For example, if the response gets a 204 (`No Content`), or if the response contains invalid JSON, attempting `r.json()` raises `requests.exceptions.JSONDecodeError`. This wrapper exception provides interoperability for multiple exceptions that may be thrown by different python versions and json serialization libraries.

via [docs](https://docs.python-requests.org/en/latest/user/quickstart/#json-response-content).